content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,313 @@
+
+You can verify that the EtcdReplicator resource is working as expected by reconfiguring `sensuctl` to communicate with the `alpha` and `beta` clusters and issuing the `sensuctl cluster-role-binding list` command. You should see 
+
+### Step 5 Register clusters
+
+In order to become visible in the web UI, a cluster must be registered. Each registered cluster must have a name and a list of one or more cluster member URLs corresponding to the backend REST API.
+
+#### Register a single cluster
+
+With `sensuctl` configured for the `gateway` cluster, run `sensuctl create` on the yaml or JSON below to register cluster `alpha`:
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+api_version: federation/v1
+type: Cluster
+metadata:
+  name: alpha
+spec:
+  api_urls:
+  - https://sensu.alpha.example.com:8080
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "Cluster",
+  "metadata": {
+    "name": "alpha"
+  },
+  "spec": {
+    "api-urls": [
+      "https://sensu.alpha.example.com:8080"
+    ]
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+#### Register additional clusters
+
+With `sensuctl` configured for `gateway` cluster, run `sensuctl create` on the yaml or JSON below to register an additional cluster and define the name as `beta`:
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+api_version: federation/v1
+type: Cluster
+metadata:
+  name: beta
+spec:
+  api_urls:
+  - https://sensu.beta.example.com:8080
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "Cluster",
+  "metadata": {
+    "name": "beta"
+  },
+  "spec": {
+    "api-urls": [
+      "https://sensu.alpha.example.com:8080"
+    ]
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+_NOTE: When logging into the `gateway` cluster web UI, any namespaces, entities, events and other resources specific to that cluster will be labled as `local-cluster`._
+
+### Step 6 Get a unified view of all your clusters in the web UI
+
+After you create clusters using the federation API, you can log in to the `gateway` Sensu web UI to view them as either the `federation-viewer` user. Switching between namespaces across federated clusters is accomplished using namespace switcher, as shown in the animation below:
+
+<img alt="animated demonstration of federated views in Sensu Web UI" title="Cross-cluster visibility in the Sensu Web UI" src="/images/federation-switcher-animated.gif" width="800 px">
+
+As the `federation-viewer` user is granted only permissions provided by the built-in `view` role, this user should be able to view all resources across all clusters, but not make any changes. Provided you've not changed the permissions of the default `admin` user, that user should be able to view, create, delete and update resources across all clusters as well.
```

Comment:
```suggestion
Because the `federation-viewer` user is granted only permissions provided by the built-in `view` role, this user should be able to view all resources across all clusters but should not be able to make any changes. If you haven't changed the permissions of the default `admin` user, that user should be able to view, create, delete, and update resources across all clusters.
```